doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-rule-based-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='test-scenarios-coverage-report-rule-based-proc']
+= Generating coverage reports for rule-based test scenarios
+
+In rule-based test scenarios, the *Coverage Report* tab contains the detailed information about the number of available rules, number of fired rules, percentage of fired rules, how many times each rule has been fired, and which rules are executed for each defined test scenario in the table. The percentage of the executed rules is also represented as a pie chart in the same *Coverage Report* tab.
```

Comment:
Suggestion - same as above.
Question - should "fired" or "executed" be used? I see both are used in this paragraph.

"In rule-based test scenarios, the *Coverage Report* tab contains information about the following:

- Number of available rules
- Number of executed rules
- Percentage of executed rules
- Percentage of executed rules represented as a pie chart
- Number of times each rule has executed
- Which rules are executed for each defined test scenario"